tile.scss

```diff
@@ -2,13 +2,11 @@ $tileColors: #7300de, #ffbf00, #ff1e00, #18a100, #0085de;
 //#fcfc03, #77fc03, #fc03f0, #03fcf4, #a103fc;
 
 .tile {
-    max-height: 32px;
-    min-height: 32px;
-    max-width: 32px;
-    min-width: 32px;
+    box-sizing: border-box;
+    width: 66px;
+    height: 66px;
```

Comment:
this makes it much easier to reason about.

before, you had `32px width + 15px padding + 15px padding + 2px border + 2px border = 66px` computed width.
with `box-sizing: border-box`, your computed width should always end up being whatever you set `width` to.